dags/utils/dataproc.py

```diff
@@ -44,6 +44,10 @@ def __init__(self,
                  gcp_conn_id='google_cloud_airflow_dataproc',
                  artifact_bucket='moz-fx-data-prod-airflow-dataproc-artifacts',
                  storage_bucket='moz-fx-data-prod-dataproc-scratch',
+                 master_disk_type='pd-standard',
+                 master_disk_size=1024,
```

Comment:
is this necessary as a default for all dataproc clusters launched by airflow? I would rather see a default of None here and set them if provided, that way other current job behavior doesn't change

Reply:
That seems reasonable - defaults are defined in the `DataprocClusterCreateOperator` already - I'm just duplicating those settings from :  https://github.com/mozilla/telemetry-airflow/blob/master/dags/operators/moz_dataproc_operator.py#L197

I've made the patch in 24dc037

Reply:
ah this was the default, my mistake.